### PR TITLE
add context to forms

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* it looks bigger than it is, sensible defaults are everywhere and will help with GA tracking
* added ‘?product=<name>’ to every contact-us form
* update server, container contact us to be more  contextual
* moved maas to use server/contact-us, not /cloud anymore

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
  - [/about/contact-us](http://0.0.0.0:8001/about/contact-us)
  - [/cloud/openstack](http://0.0.0.0:8001/cloud/openstack)
  - [/cloud/partners](http://0.0.0.0:8001/cloud/partners)
  - [/cloud/plans-and-pricing](http://0.0.0.0:8001/cloud/plans-and-pricing)
  - [/cloud/storage](http://0.0.0.0:8001/cloud/storage)
  - [/cloud/training](http://0.0.0.0:8001/cloud/training)
  - [/containers/contact-us](http://0.0.0.0:8001/containers/contact-us)
  - [/containers/docker-ubuntu](http://0.0.0.0:8001/containers/docker-ubuntu)
  - [/containers/](http://0.0.0.0:8001/containers/)
  - [/containers/kubernetes](http://0.0.0.0:8001/containers/kubernetes)
  - [/containers/lxd](http://0.0.0.0:8001/containers/lxd)
  - [/core](http://0.0.0.0:8001/core)
  - [/desktop/education](http://0.0.0.0:8001/desktop/education)
  - [/desktop/enterprise](http://0.0.0.0:8001/desktop/enterprise)
  - [/desktop/government](http://0.0.0.0:8001/desktop/government)
  - [/desktop/partners](http://0.0.0.0:8001/desktop/partners)
  - [/download/cloud/autopilot](http://0.0.0.0:8001/download/cloud/autopilot)
  - [/download/cloud/](http://0.0.0.0:8001/download/cloud/)
  - [/download/server/arm](http://0.0.0.0:8001/download/server/arm)
  - [/internet-of-things](http://0.0.0.0:8001/internet-of-things)
  - [/search](http://0.0.0.0:8001/search)
  - [/server/contact-us](http://0.0.0.0:8001/server/contact-us)
  - [/server/hyperscale](http://0.0.0.0:8001/server/hyperscale)
  - [/server/](http://0.0.0.0:8001/server/)
  - [/server/maas](http://0.0.0.0:8001/server/maas)
  - [/server/management](http://0.0.0.0:8001/server/management)
  - [/support/contact-us](http://0.0.0.0:8001/support/contact-us)
  - [/support/esm](http://0.0.0.0:8001/support/esm)
  - [/support](http://0.0.0.0:8001/support)
  - [/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- click on any contact us page and see that it goes to one.

### CODE REVIEW

File: templates/cloud/partners.html [hunk 1]
```diff
@@ -89,7 +89,7 @@ <h2 class="p-heading--muted u-align--center">A selection of our cloud partners</
     <div class="col-8">
       <h2>Become a cloud partner</h2>
       <p>To learn more about becoming a charm, Ubuntu certified public cloud or Ubuntu OpenStack partner, please contact us today.</p>
-      <p><a href="http://partners.ubuntu.com/contact-us" class="p-button--brand">Get in touch</a></p>
+      <p><a href="http://partners.ubuntu.com/contact-us?product=partners" class="p-button--brand">Get in touch</a></p>
```
Comment (anthonydillon) [inline templates/cloud/partners.html]: Do you need this on partners forms?

File: templates/cloud/storage.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
       <div class="col-6">
         <h1>Ubuntu&nbsp;Advantage&nbsp;Storage</h1>
         <p>Ubuntu Advantage Storage from Canonical embeds the proven software&dash;defined storage (<abbr title="Software&dash;defined storage">SDS</abbr>) technologies of Ceph, NexentaEdge, Swift and SwiftStack, into a 24x7&dash;supported software solution with a unique metered pricing model.</p>
-        <p><a class="p-button--brand" href="/cloud/contact-us?product=storage">Request a demo</a></p>
+        <p><a class="p-button--brand" href="/cloud/contact-us?product=cloud-storage">Request a demo</a></p>
```
Comment (anthonydillon) [inline templates/cloud/storage.html]: The form is expecting `storage` not `cloud-storage`

File: templates/cloud/storage.html [hunk 3]
```diff
@@ -104,7 +104,7 @@ <h3 class="p-matrix__title">SwiftStack</h3>
                 <p>Legacy storage solutions are often priced per&dash;node or according to total disk capacity. This can lead to a penalty for good practices such as high durability&nbsp;replication.</p>
                 <p>Ubuntu Advantage Storage is different. Our pricing is based on the amount of data you record in your storage pool, regardless of how it is written or how much disk space is used.</p>
                 <p>Ubuntu Advantage Storage means you don&rsquo;t have to pay for replicas or empty space.</p>
-                <p><a href="/cloud/contact-us?product=storage">Find out how much you can save&nbsp;&rsaquo;</a></p>
+                <p><a href="/cloud/contact-us?product=cloud-storage">Find out how much you can save&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/cloud/storage.html]: Same here. Either update the contact-us if statement or change this to just storage.

File: templates/cloud/storage.html [hunk 4]
```diff
@@ -124,7 +124,7 @@ <h3 class="p-matrix__title">SwiftStack</h3>
                   <li class="p-list__item is-ticked">World&dash;class support from Canonical, experts in large&dash;scale, multi&dash;site&dash;replicated deployments.</li>
                   <li class="p-list__item is-ticked">Built with Ubuntu {{ lts_release_full }} with five years of support for every component in the stack.</li>
                 </ul>
-                <p><a class="p-button--neutral" href="/cloud/contact-us?product=storage">Request a demo</a></p>
+                <p><a class="p-button--neutral" href="/cloud/contact-us?product=cloud-storage">Request a demo</a></p>
```
Comment (anthonydillon) [inline templates/cloud/storage.html]: Same here.

File: templates/download/cloud/autopilot.html [hunk 5]
```diff
@@ -231,7 +231,7 @@ <h3 class="p-list-step__title"><span class="p-list-step__bullet">12</span>Monito
           <div class="col-12">
             <h2>Need more help?</h2>
             <p>Let our cloud experts help you take the next step.</p>
-            <p><a href="/cloud/contact-us">Contact us&nbsp;&rsaquo;</a></p>
+            <p><a href="/cloud/contact-us?product=autopilot">Contact us&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/download/cloud/autopilot.html]: `cloud-autopilot`?

File: templates/shared/contextual_footers/_cloud_bootstack.html [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <div class="col-4 p-divider__block">
   <h3 class="p-heading--four">Want a fully managed private cloud?</h3>
   <p>BootStack is your OpenStack private cloud with our experts responsible for design, deployment and availability.</p>
-  <p><a href="/cloud/openstack/managed-cloud/contact-us" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'BootStack', 'eventLabel' : 'Want a fully managed private cloud?', 'eventValue' : undefined });">Get BootStack&nbsp;&rsaquo;</a></p>
+  <p><a href="/cloud/contact-us?product=contextual-footer-managed-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'BootStack', 'eventLabel' : 'Want a fully managed private cloud?', 'eventValue' : undefined });">Get BootStack&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/shared/contextual_footers/_cloud_bootstack.html]: Shouldn't this just be `managed-cloud` as there is no option for `contextual-footer-managed-cloud`.

File: templates/cloud/partners.html (hunk 1 above)
Comment (pmahnke) [inline templates/cloud/partners.html]: no, no I do not...

File: templates/cloud/storage.html (hunk 2 above)
Comment (pmahnke) [inline templates/cloud/storage.html]: I meant to update the form... and managed-cloud is now cloud-managed-cloud as well.

File: templates/shared/contextual_footers/_cloud_bootstack.html (hunk 6 above)
Comment (pmahnke) [inline templates/shared/contextual_footers/_cloud_bootstack.html]: well, I was using defaults here, but I fixed it up to use the context where available

**Pull request merged**